### ISSUE TITLE
Accessibility - iOS - New Icon has Color Contrast Issues

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#98]
When the newly updated Sweat and Toil application updated on our iPad (iOS), it was difficult to see on the original default background screen. The mostly dark blue image on the night sky background made it hard to see because of poor color contrast with the black starry night sky.

Please see the attached image.  You could have an issue with another image if the person has a light colored image, and so is their background.

![sweat and toil - new icon - difficult to see depending on where it s palced on background](https://user-images.githubusercontent.com/10423487/44361193-1edcdb00-a48b-11e8-81fe-6d5849b1d81b.png)

**Comment by jfc3-dol** [comment]
On darker background screens the icon has poor color contrast.

**Comment by clguidi** [comment]
John, we now have a new permanent icon which should address this. Please check and close this issue if possible. Thank you.

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,

Looking through old mail in the "Other" tab, I noticed this issue was there and still open.

The new Sweat and Toil iOS icon is better, so I'm going to close this issue.
